website/static/js/fangorn.js

```diff
@@ -479,6 +481,11 @@ function doItemOp(operation, to, from, rename, conflict) {
         }
         // no need to redraw because fangornOrderFolder does it
         orderFolder.call(tb, from.parent());
+
+        if (operation === OPERATIONS.MOVE) {
```

Comment:
It is brittle to put the responsibility of updating comments' root targets on the front end. Important business rules like this should be handled by the backend application. In other words, when files get moved, the backend should handle the Comment updates without the frontend having to do extra work.